app/assets/stylesheets/posts.scss

```diff
@@ -15,6 +15,10 @@ pre.post-body-pre-block {
   word-wrap: break-word;       /* Internet Explorer 5.5+ */
 }
 
+blockquote {
+	font-size: inherit;
```

Comment:
Can we make the indent the same 2 spaces as everything else, rather than a tab?

Reply:
Oops sorry. I have `autocmd FileType css,scss set ts=2 sts=2 sw=2 et` in my editor config (tab is one less character than spaces).